integration-test/src/test/java/com/klaytn/caver/model/dto/Transaction.java

```diff
@@ -93,10 +93,14 @@ public BigInteger getNonce() {
         }
 
         public BigInteger getGasPrice() {
-            if (gasPrice == null) return DEFAULT_GAS_PRICE;
```

Comment:
Is there any other place where DEFAULT_GAS_PRICE is used?

If not, it could be good to remove its declaration.

Reply:
That is used in the before common architecture features.
I tried to remove that, but that will affect to compatibility.
In the common architecture, if user uses caver-java like how we guided, then gas price is set  by fetching from the Node.

Reply:
DEFAULT_GAS_PRICE is just a variable which is set as a static value, and the value is changed several times (25 -> 750 -> 250). If we guide user to fetch the value via API, do we need to leave this as it is?

I was talking about the declaration of DEFAULT_GAS_PRICE because it is used only once here.

Reply:
`DEFAULT_GAS_PRICE` is for now still 25 Ston, and is used with before common architecture features.

I wanted to delete that value, but it was used in the test code of caver-java until recently (Denver and i removed those), so I left it because users using caver-java may also be using the usability of using DEFAULT_GAS_PRICE.
If user uses Baobab and Cypress with DEFAULT_GAS_PRICE, an error is occurred from the time the gas price is changed in the network.
But if user configures their network, and not change the gas price, then this can be used without errors.

And the DEFAULT_GAS_PRICE value is not updated. 
After the common architecture usage fetching the gas price or base fee from the Node I mentioned above, DEFAULT_GAS_PRICE is not used at all.

Reply:
Okay. I see. Let's leave this as it is, then. Thanks!

Reply:
Oh wait sorry, I confused with the other default gas price value. I can remove this!

Reply:
I removed this value !! I confused with `GAS_PRICE_25_STON`, which is renamed from `DEFAULT_GAS_PRICE`.

Reply:
got it Thanks.